packages/frontier-libraries/stdlib/tsconfig.json

```diff
@@ -1,7 +1,28 @@
 {
   "extends": "../../../tsconfig.base.json",
+  "compilerOptions": {
+    "baseUrl": "src",
+    "paths": {
+      "@/*": [
+        "src/*"
+      ]
+    },
+    "declaration": true,
```

Comment:
Dont override options from tsconfig.base.json. Once the value exists in the base file do not create a duplicate entry in this file. The typescript compiler merges the two files which creates unnecessary redundancy

Reply:
Unnderstood